src/client/components/PlaceInfo/PlaceInfo.jsx

```diff
@@ -98,52 +65,54 @@ class PlaceInfo extends Component {
         dialogClassName="placeinfo"
       >
         <Modal.Header closeButton>
-          <Modal.Title className="info-title">{PLACE.city}</Modal.Title>
+          <Modal.Title className="info-title">{place.city}</Modal.Title>
         </Modal.Header>
         <Modal.Body>
-          <div className="info-body">
-            <div className="image-row">
-              {/* {_.map(PLACE.images, (image, key) => {
-                return <img key={key} className="image" src={image} />;
-              })} */}
-              {renderImages(PLACE, isMobile)}
-            </div>
-            <div className="info-content">
-              <h1 className="name">{PLACE.name}</h1>
-              <p className="description">{PLACE.description}</p>
-              <p className="location">
-                <span className="word">Location: </span>
-                {PLACE.address}
-              </p>
-              {/*<div className="review-container">
-                <p className="section-title">Reviews:</p>
-                {_.map(PLACE.reviews, (review, key) => {
-                  return (
-                    <div className="review" key={key}>
-                      <p className="message">"{review.message}"</p>
-                      <p className="reviewer">
-                        - {review.reviewer.name}, {review.reviewer.status}
-                      </p>
-                    </div>
-                  );
-                })}
-              </div>*/}
+          { _.isEmpty(place) ? <Preloader /> :
+            <div className="info-body">
+              <div className="image-row">
+                {/* {_.map(place.images, (image, key) => {
+                  return <img key={key} className="image" src={image} />;
+                })} */}
+                {place.images && renderImages(place.images, isMobile)}
+              </div>
+              <div className="info-content">
+                <h1 className="name">{place.name}</h1>
+                <p className="description">{place.description}</p>
+                <p className="location">
+                  <span className="word">Location: </span>
+                  {place.address}
+                </p>
+                {/*<div className="review-container">
```

Comment:
Can we remove this commented code?

Reply:
I left it here because it will be useful when we have review data. And I think we should try to get review data because it will be very useful for the users. But I don't know how hard it will be to do that. Maybe can try the Yelp API.